### DISCUSSION TITLE
About the "docsettings" folder.

### DISCUSSION THREAD

**Discussion by ichnilatis-gr** [koreader/koreader#10145] [Q&A]
After the creation of the "docsettings" folder for the .sdr files, I would like to ask if we have to manually transfer the already created .sdr files of the documents we are reading there.

Thanks.

**Comment by mergen3107** [comment]
No, you don’t need to do anything. The sdr folder is transferred by itself when open a book and close (or by timer, depending on your metadata saving setting in top menu) in the new KOReader for the first time.

**Comment by ichnilatis-gr** [comment]
What about the sdr folders of documents that I have read and I'm not going to open again –at least not soon?

**Comment by hius07** [comment]
I am afraid manual transferring of sdr folders is not an easy task (you may create a wrong path).
To transfer automatically you can: open, rename, move, or change status of a book.

**Comment by hius07** [comment]
sdr folders along the books are supported (forever), don't worry about them.

**Comment by ichnilatis-gr** [comment]
I would just like to empty my folders of sdr folders so that I can more easily navigate to my files in the Pocketbook library...

**Comment by hius07** [comment]
No group migration yet, sorry.

**Reply by ichnilatis-gr** [reply]
Never mind.

**Reply by hius07** [reply]
But hope will be: https://github.com/koreader/koreader/pull/10149

**Reply by ichnilatis-gr** [reply]
Oh, @hius07, you are willing and able to meet many needs that come up and we thank you. Only [one of my requests](https://github.com/koreader/koreader/issues/9900) still remains unfulfilled... 😉
